marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/GenerateFunctionMetadataCommand.java

```diff
@@ -0,0 +1,81 @@
+package com.marklogic.hub.deploy.commands;
+
+import com.marklogic.appdeployer.command.AbstractCommand;
+import com.marklogic.appdeployer.command.CommandContext;
+import com.marklogic.appdeployer.command.SortOrderConstants;
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.datamovement.ApplyTransformListener;
+import com.marklogic.client.datamovement.DataMovementManager;
+import com.marklogic.client.datamovement.QueryBatcher;
+import com.marklogic.client.document.ServerTransform;
+import com.marklogic.client.query.StructuredQueryBuilder;
+import com.marklogic.hub.DatabaseKind;
+import com.marklogic.hub.HubConfig;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.concurrent.TimeUnit;
+
+@Component
+public class GenerateFunctionMetadataCommand extends AbstractCommand {
+    @Autowired
+    private HubConfig hubConfig;
+    private boolean forceLoad = true;
+    private Throwable caughtException;
+
+    public GenerateFunctionMetadataCommand() {
+        super();
+        // This command has to be executed after modules are loaded
+        setExecuteSortOrder(SortOrderConstants.LOAD_MODULES + 1);
+    }
+    public void setForceLoad(boolean forceLoad) {
```

Comment:
I recommend dropping this feature. I know we have it on commands for loading modules, but I don't see a use case for this for function metadata - for now, I think we always want to generate function metadata to ensure it's up-to-date.